docs/getting-started-guides/photon-controller.md

```diff
@@ -0,0 +1,232 @@
+---
+---
+
+The example below creates a Kubernetes cluster using VMware's Photon
+Controller. The cluster will have one Kubernetes master and three
+Kubernetes nodes.
+
+* TOC
+{:toc}
+
+### Prerequisites
+
+1. You need administrator access to a [VMware Photon
```

Comment:
(I made the changes you requested, other than the table--I need clarification on that.)

I think it's outside the scope of this document: it's a big piece of software to install, and the installation is non-trivial. 

Elsewhere there is a link to the github page for Photon Controller. It has some documentation as well as downloads for releases. I have a movie of the kube-up process that I made for internal purposes: 
https://www.youtube.com/watch?v=NhRdONwV0U0